composer.json

```diff
@@ -8,14 +8,23 @@
   "minimum-stability": "stable",
   "autoload": {
     "psr-4": {
-      "": "src/",
+      "Google\\Photos\\": "src/Google/Photos/",
+      "GPBMetadata\\Photos\\": "src/GPBMetadata/Photos/"
```

Comment:
Ah, I had some trouble running this -  this needs to include `Google` in the name and path:
```
      "GPBMetadata\\Google\\Photos\\": "src/GPBMetadata/Google/Photos/"
```
(This should be last change, then we can merge it in. Appreciate you working through the comments!)

Reply:
whoops

Reply:
resolved in 1ee0a3e